src/passim-integration-test.py

```
@@ -0,0 +1,324 @@
+            f"passimd did not start listening on port {cls.DAEMON_PORT}"
+        )
+
+    def _https_get(self, path, expected_status=200):
+        """Make an HTTPS GET request to the daemon."""
+        ctx = ssl.create_default_context()
+        ctx.check_hostname = False
+        ctx.verify_mode = ssl.CERT_NONE
+        url = f"https://127.0.0.1:{self.DAEMON_PORT}{path}"
+        req = urllib.request.Request(url)
+        try:
+            resp = urllib.request.urlopen(req, context=ctx)
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)